core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/http/pipeline/stages/HttpChecksumStage.java

```diff
@@ -67,6 +68,11 @@ public SdkHttpFullRequest.Builder execute(SdkHttpFullRequest.Builder request, Re
             return request;
         }
 
+        // Check for SRA code-path
```

Comment:
So we'll still need to keep this class to add md5 checksum header (after SRA is fully released)?